### DISCUSSION TITLE
[Feature Request] Option to stack graphql documentation (schema) instead of overlaying

### DISCUSSION THREAD

**Discussion by terion-labs** [Kong/insomnia#3740] [Ideas]
**Is your feature request related to a problem? Please describe.**
Working with complex graphql apis requires  a lot of documentation exploration. Currently docs window overlays query results which forces to open/close it regullary. Very frustrating

**Describe the solution you'd like**
Options to stack documentation window in third column (like in graphiql) and option to enable it by default

**Comment by elemanhillary-zz** [comment]
@terion-name i have got yo back, say no worries
